File: colorMixer.java
package colormixingSystem;

import java.util.Scanner;

public class colorMixer {

	public static void main(String[] args) {
		
		String red = "red";
		String blue = "blue";
		String yellow = "yellow";
		
		Scanner input = new Scanner(System.in);
		System.out.println("Enter primary color (red, blue, yellow):");
		String p1 = input.nextLine();
		System.out.println("Enter another primary color(red, blue, yellow):");
		String p2 = input.nextLine();
		
		
		
		if( (p1.equals(red)) & (p2.equals(blue))) {
			System.out.println("When you mix red and blue, you get purple");
		}
		else if( (p1.equals(red)) & (p2.equals(yellow))) {
			System.out.println("When you mix red and yellow, you get orange");
		}
		else if( (p1.equals(red)) & (p2.equals(red))) {
			System.out.println("When you mix red and red, you get red");
		}
		else if((p1.equals(blue)) & (p2.equals(red))) {
			System.out.println("When you mix blue and red, you get purple");
		}
		else if( (p1.equals(blue)) & (p2.equals(yellow))) {
			System.out.println("When you mix blue and yellow, you get green");
		}
		else if ( (p1.equals(blue)) & (p2.equals(blue))) {
			System.out.println("When you mix blue and blue,you get blue");
		}
		
		else if ((p1.equals(yellow)) & (p2.equals(red))) {
			System.out.println("When you mix yellow and red, you get orange");
			
		}
		else if (p1.equals(yellow) & (p2.equals(blue))) {
			System.out.println("When you mix yellow and blue, you get green;");
		}
		else if (p1.equals(yellow) & (p2.equals(yellow))) {
			System.out.println("When you mix yellow and yellow, you get yellow");
		}
		else {
			System.out.println("You didn't input two primary colors");
		}
	

	

		

	}

}

File: color_mixer.py
primaries = ['red', 'yellow', 'blue']
answer = input('Enter a primary color:').lower().strip()
if answer in primaries:
    print('You entered', answer1)
else:
    while answer1 not in primaries:
        print("Sorry, that's not a primary color. Please try again.")
        answer1 = input('Enter a primary color:').lower() .strip()
    print('You entered', answer1)

answer2 = input('Enter a second primary color:').lower().strip()
while answer2 == answer1 or answer1 not in primaries:
    if answer2 not in primaries:
        print("Sorry, that's not a primary color. Try again.")
        answer2 = input('Enter another primary').lower() .strip()
    else:
        print('That was the same color as before. Please try again')
        answer2 = input('Enter a different primary color.').lower() .strip()
if answer2 in primaries:
    print('You entered', answer2)

mixture = answer1 + answer2

secondaries = {
    'redyellow' : 'orange',
    'yellowred' : 'orange',
    'redblue' : 'purple',
    'bluered' : 'purple',
    'yellowblue' : 'green',
    'blueyellow' : 'green'
}

print(answer1, 'mixed with', answer2, 'turns into', secondaries[mixture])
